Sources/BraveWallet/Extensions/SolanaInstruction+Extensions.swift

```diff
@@ -37,6 +37,18 @@ extension BraveWallet.SolanaInstruction {
     }
     return Strings.Wallet.solanaUnknownInstructionName
   }
+  
+  /// Returns the `to_account` pubkey for the instruction if available
+  var toPubkey: String? {
+    guard let index = decodedData?.accountParams.firstIndex(where: { $0.name == "to_account" }) else { return nil }
+    return accountMetas[safe: index]?.pubkey
```

Comment:
Could there be an edge case where `to_account` exists but its `pubkey` doesn't? I tried to test it dynamically by deleting `transaction.instructions[0].keys[0].pubkey` from the transaction body, but have a hard time manually crafting Solana transactions. Maybe we should check if `pubkey` exists in `guard` close just in case.

Reply:
There is a 1:1 relationship between `accountMetas` and `decodedData.accountParams` so this shouldn't occur, but _if_ there was no account available then `accountMetas[safe: index]?` would fail to unwrap and this computed property would return nil so we would only display the from address & blockie in the transaction header.

The `safe:` helper is found here (if the index does not exist, return nil): https://github.com/brave/brave-ios/blob/development/Sources/Shared/Extensions/ArrayExtensions.swift#L57-L62